crates/atuin/src/command/external.rs

```diff
@@ -54,7 +54,11 @@ fn render_not_found(subcommand: &str, bin: &str) -> StyledStr {
     let _ = write!(output, "{error}error:{error:#} ");
     let _ = write!(
         output,
-        "unrecognized subcommand '{invalid}{subcommand}{invalid:#}' "
+        "{} ",
```

Comment:
This is an awkward example but maybe helps show where extra steps are required (e.g. a simple dynamic string would be wrapped by `t!(..)` and done)